### PR TITLE
DEV: LEAF-1585 - OR Operator for Reports and Search

### PR DESCRIPTION
**Pull request by pete-nerantzinis**

### CODE REVIEW

**Review by Vaceek — APPROVED** [review]

**Review by gsteinhoff — CHANGES_REQUESTED** [review]
Just a small change in logic but otherwise looks great.

File: LEAF_Request_Portal/form.php [hunk 1]
```diff
@@ -2487,6 +2488,23 @@ public function query($inQuery)
...
+            // Logic for AND/OR Operator's
+            $op = 'AND';
+            if ($q['op']) {
+                $op = $q['op'];
+            }
+            if ($count === 0) {
+                $op = '';
+                $conditions = '(';
+            }
+            if ($count != 0) {
```
Comment (gsteinhoff) [inline LEAF_Request_Portal/form.php:2500]: The logic here can be consolidated into count is zero or else.  don't need to check it again if it's just every other number but zero.

**Review by gsteinhoff — CHANGES_REQUESTED** [review]

File: LEAF_Request_Portal/form.php (hunk 1 above)
Comment (pete-nerantzinis) [reply LEAF_Request_Portal/form.php]: Consolidated 👍

**Review by gsteinhoff — APPROVED** [review]
Looking good, nice work!

**Review by aerinkayne — APPROVED** [review]

**Review by Pelentan — APPROVED** [review]

**Pull request merged**